test/extended/oauth/groupsync.go

```diff
@@ -1,111 +1,419 @@
 package oauth
 
 import (
-	"fmt"
+	"bufio"
+	"bytes"
+	"context"
+	"encoding/json"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path"
+	"strings"
 
 	g "github.com/onsi/ginkgo"
 	o "github.com/onsi/gomega"
+	"gopkg.in/ldap.v2"
+	"sigs.k8s.io/yaml"
 
-	"time"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kdiff "k8s.io/apimachinery/pkg/util/diff"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/kube-openapi/pkg/util/sets"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	userv1 "github.com/openshift/api/user/v1"
+	userclient "github.com/openshift/client-go/user/clientset/versioned"
+	"github.com/openshift/library-go/pkg/security/ldapclient"
 
 	"github.com/openshift/origin/test/extended/testdata"
 	exutil "github.com/openshift/origin/test/extended/util"
-	"github.com/openshift/origin/test/extended/util/image"
-	"k8s.io/apimachinery/pkg/util/wait"
-	e2e "k8s.io/kubernetes/test/e2e/framework"
 )
 
+func init() {
+	utilruntime.Must(userv1.Install(scheme))
```

Comment:
I'd suggest to move this initialization to `test/extended/oauth/htpasswd.go` where this is created since this will affect that `scheme` variable. Alternatively, I'd encourage you to create your own just for this test.

Reply:
looks like I only used this during development, I'll remove it